### PR TITLE
add maintainers wanted announcement in readme

### PR DESCRIPTION
**Pull request by edgarriba**
The project maintainers we don't have time to keep the project alive. We are looking for people to lead the development. Many tasks like xtensor and gpu integration and started during the last gsoc 2017 are still pending to be finished.

**If anyone is interested or have new ideas/proposals just write it down here so that we can have further discussion.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 <div align="center">
   <img src="https://github.com/tiny-dnn/tiny-dnn/blob/master/docs/logo/TinyDNN-logo-letters-alpha-version.png"><br><br>
 </div>
+
+-----------------
+
+[![Maintainers Wanted](https://img.shields.io/badge/maintainers-wanted-red.svg)](https://github.com/pickhardt/maintainers-wanted)
+
+## The project may be abandoned since the maintainer(s) are just looking to move on. In the case anyone is interested in continuing the project, let us know so that we can discuss next steps.
```
Comment (bhack) [inline README.md]: Where? In a new ticket? What are the active communication channel?

Comment (edgarriba) [reply README.md]: @bhack right, I will open a ticket in order to point there

**Comment by utilForever** [comment]
Hello, I try to understand this code by rewriting it from scratch.
I want to keep developing the project when it reaches the same level.
https://github.com/utilForever/CubbyDNN

**Comment by edgarriba** [comment]
@utilForever starting the project from scratch was one of the discussion points some time ago. Please, visit https://groups.google.com/forum/#!forum/tiny-dnn-dev and leave some comments because you might consider some of the stuff reported there if you want to start this thing from scratch. Looks a good initiative though.

**Comment by utilForever** [comment]
@edgarriba Thanks. I visited google groups and read "Project cleanup". However, I don't have permission to write comments. Anyway, Can I start this project?

**Pull request merged**